src/App.vue

```diff
@@ -1,24 +1,31 @@
 <template>
   <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png" />
-    <HelloWorld msg="Welcome to Your Vue.js App" />
+    <HomePage></HomePage>
+    <RegisterPage></RegisterPage>
```

Comment:
We don't need to place our Pages in the App 😄 
Vue Router will do it for us and render the page that corresponds to current path in the place where `router-view` component is used.